login/style.css

```diff
@@ -1,4 +1,73 @@
-#login-image {
-  width: 100px;
-  height: 100px;
-}
\ No newline at end of file
+
+body {
+  background-color: aliceblue;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
This is what is called the font stack. You can learn a bit of what is a font-family and their differences.
This is based on:
https://markdotto.com/2018/02/07/github-system-fonts/
And simply suggested by my editor (vscode)